description.md

```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easier digestible format of presenting information.
+As such, it is typically used in conjunction with other content types, such as reading, slides or demos.
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
```

Comment:
Not necessarily true. `.svg` images may serve as both the output and the editable building blocks (without requiring the `.drawio` files which are mostly the same thing).

Secondly, add a line about recordings and screenshots. There's nothing editable about them and unfortunately, they are to be stored as they are.

Reply:
>Not necessarily true. .svg images may serve as both the output and the editable building blocks (without requiring the .drawio files which are mostly the same thing).

That's just a particular case where the output and the editable building block are the same file.

Reply:
> Secondly, add a line about recordings and screenshots. There's nothing editable about them and unfortunately, they are to be stored as they are.

I will elaborate on this.

Reply:
I would word it as "being contributable / editable" - where possible (screenshots can't fit this). Such that someone could then get the media content and be able to improve it according to the 5R content properties. Only then, I would mention storing both the original content and the output (where possible).